pkg/mimir/mimir.go

```diff
@@ -225,6 +225,10 @@ func (c *Config) Validate(log log.Logger) error {
 	if err := c.Querier.Validate(); err != nil {
 		return errors.Wrap(err, "invalid querier config")
 	}
+	if c.Querier.EngineConfig.Timeout > c.Server.HTTPServerWriteTimeout {
+		return fmt.Errorf("querier timeout (%s) must be lower than HTTP server write timeout (%s)",
```

Comment:
```suggestion
		return fmt.Errorf("querier timeout (%s) must be lower than or equal to HTTP server write timeout (%s)",
```

Reply:
Good catch! 
Thanks for your review!  💚 💙 💜 💛 ❤️